x-pack/plugins/canvas/public/components/workpad_header/share_menu/share_menu.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { i18n } from '@kbn/i18n';
+import { State } from '../../../../types';
+import { usePlatformService, useReportingService } from '../../../services';
+import { getPages, getWorkpad } from '../../../state/selectors/workpad';
+import { useDownloadWorkpad } from '../../hooks';
+import { ShareMenu as ShareMenuComponent } from './share_menu.component';
+
+const strings = {
+  getUnknownExportErrorMessage: (type: string) =>
+    i18n.translate('xpack.canvas.workpadHeaderShareMenu.unknownExportErrorMessage', {
+      defaultMessage: 'Unknown export type: {type}',
+      values: {
+        type,
+      },
+    }),
+};
+
+export const ShareMenu: FC = () => {
```

Comment:
nit: if you're not using props and there's no use for `children`, you can leave this off and have no type annotation.